lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -7,6 +7,10 @@
 #ifndef TEE_API_DEFINES_EXTENSIONS_H
 #define TEE_API_DEFINES_EXTENSIONS_H
 
+/* RSA signatures with MD5 hash */
```

Comment:
I thin this should explain why these magic numbers:
`/* Value prefixed with vendor ID bit31 with by TEE bitfields IDs */`

Reply:
Applied, Thank you